### DISCUSSION TITLE
Can't delete branch for PR merged from fork

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#135823] [Mobile]
I got a notification that https://github.com/github/docs/pull/34305
Was merged

![Screenshot_20240814-174147.png](https://github.com/user-attachments/assets/164a2da5-5da1-4dcf-94e0-d8971d51b8f2)

**Comment by taki-on** [comment]
@jsoref Thank you for sharing it. Deleting branch is not supported yet on mobile. Let me create an issue to discuss it with the team :)

**Reply by jsoref** [reply]
It's offered _some_ of the time. I've used it.

**Reply by taki-on** [reply]
oops you are right 🤦 We support deleting branch when a PR is merged. However, it looks like we do not support deleting a branch in a forked repo.

**Reply by jsoref** [reply]
I appreciate that humility. I'm currently frustrated by a different org which lacks that humility/awareness...
